gregoriotex/details.html

```diff
@@ -577,7 +577,7 @@
 <li><code>\gresetlines{invisible}</code> completely removes the staff lines.</li>
 <li><code>\gresetclef{invisible}</code> removes the clefs.</li>
 <li><code>\grechangestafflinethickness{<i>&lt;integer&gt;</i>}</code> changes the thickness of the staff lines. The default value is the same as the staff size. You can only put integers here.</li>
-<li><code>\gresetgregoriofont[op]{parmesan}</code> uses the Dominican variant of the Parmesan font for the neumes.</li>
+<li><code>\gresetgregoriofont[op]{granapadano}</code> uses the Dominican variant of the Granapadano font for the neumes.</li>
```

Comment:
I should have been more clear.  The font name is Grana Padano, this becomes `granapadano` for the filename and as the argument of the function.